### ISSUE TITLE
About transfer learning

### ISSUE THREAD

**Issue by seabluescn** [SciSharp/TensorFlow.NET#840]
Hi,
How to use Tensorflow.NET to implement code similar to the following by Python?
 
# Start
best_model=tf.keras.models.load_model("models/chapter11_q8_bestmodel.h5")
best_model.layers[0].trainable=False   

l_layer=len(best_model.layers)-1
new_model=Sequential(best_model.layers[0:l_layer])  

new_model.add(Dense(5,activation=tf.nn.softmax))
# End

Thanks!

**Comment by Oceania2018** [comment]
Please check this example: https://github.com/SciSharp/SciSharp-Stack-Examples/blob/master/src/TensorFlowNET.Examples/ImageProcessing/TransferLearningWithInceptionV3.cs
